pyorbital/tlefile.py

```diff
@@ -617,6 +611,9 @@ def table_exists(db, name):
     return db.execute(query, (name,)).fetchone() is not None  # nosec
 
 
+def _utcnow():
+    return dt.datetime.now(tz=dt.timezone.utc).replace(tzinfo=None)
```

Comment:
why do we want to remove the timezone info again?

Reply:
To make them naive. Keeping the timezone would require a lot more work I assume. For example, sanitizing user inputs.

Reply:
ok

Reply:
If the datetimes are timezone-unaware, they can't be compared with tz-aware datetimes. I thought our goal was to make everything tz-aware across the Pytroll ecosystem.

Reply:
And I think makin all of the datetimes `tzinfo=None` is even more work than make them use UTC. The `None` case would mean addinitional TZ conversion for pretty much everywhere where we declare the times as UTC.

Reply:
I don't disagree that we should/could move toward UTC-aware times everywhere, but that wasn't my goal with my changes in this PR. I was mostly concerned about the warnings generated while running tests. I understand about 2% of pyorbital and the TLE portions, on a code level, are not part of that understanding. I should not be the person who does that conversion/translation in pyorbital. Maybe I'm overestimating how much datetimes are involved. If so I can take a stab at it, but I'm still not sure this is the PR for it.